assets/src/dashboard/components/pageStructure/index.js

```diff
@@ -72,15 +79,55 @@ export const LeftRailContainer = styled.nav`
   background: ${({ theme }) => theme.colors.white};
   border-right: ${({ theme }) => theme.leftRail.border};
   z-index: ${Z_INDEX.LAYOUT_FIXED};
+  transition: transform 0.25s cubic-bezier(0.4, 0, 0.2, 1),
```

Comment:
I'm just curious, do you want to use one of the Bezier values in constants/animation for this?

Reply:
I can definitely do that.